core/trino-main/src/main/java/io/trino/execution/QueryInfo.java

```diff
@@ -79,6 +79,7 @@
     private final boolean completeInfo;
     private final Optional<ResourceGroupId> resourceGroupId;
     private final Optional<QueryType> queryType;
+    private final String retryPolicy;
```

Comment:
Why not the enum?

Reply:
IDK - I think I for some reason thought that QueryInfo is in SPI and you cannot do that. WIll change.